pkg/notify/mattermost.go

```diff
@@ -215,3 +221,15 @@ func mmShortNotification(event events.Event) []*model.SlackAttachmentField {
 		},
 	}
 }
+
+// getAllChannels return all the channels configured under Accessbinding
+func (m *Mattermost) getAllChannels() []string {
```

Comment:
The function name is quite confusing. By reading it gives a sense that it returns all channels in Mattermost namespaces. Which is not the case. Please use something like `getAllAccessChannels/getAllConfiguredChannel`